Bridges/HttpKernel.php

```diff
@@ -240,12 +238,22 @@ protected function mapResponse(HttpResponse $reactResponse, SymfonyResponse $syR
 
         $reactResponse->writeHead($syResponse->getStatusCode(), $headers);
 
-        $stdOut = '';
-        while ($buffer = @ob_get_clean()) {
-            $stdOut .= $buffer;
+        // asynchronously get content
+        ob_start(function($buffer) use ($reactResponse) {
+            $reactResponse->write($buffer);
```

Comment:
> but output buffer is not added to final response

Could you elaborate what that means? Which OS and which PHP version? I've had success testing this on Debian Jessie, php 5.6.